apps/i18n/common/en_us.json

```diff
@@ -1904,6 +1904,10 @@
   "plLandingHeading": "My Professional Learning",
   "plLandingSubheading": "Submit your feedback",
   "plLandingStartSurvey": "Start survey",
+  "plLandingGettingStartedHeading": "Getting started with Professional Learning",
+  "plLandingGettingStartedSubHeading": "Code.org Professional Learning",
+  "plLandingGettingStartedDescription": "Help today's students become tomorrow's superheroes with professional learning that meets your needs. We offer different kinds of professional learning to meet your needs.",
+  "plLandingGettingStartedButton": "Learn about professional learning",
```

Comment:
just checking -- is it intentional that "Professional Learning" is capitalized in other strings but not this one?

Reply:
Good question 🤔 That's how it was on the [mock](https://www.figma.com/file/sZ6d1uhJWFksJ6j3Fnmbnc/PL-Flow-Improvements?type=design&node-id=1958%3A10453&mode=design&t=bI4cvDycRyj4aHQr-1). The other buttons on this mock are using the same sentence case structure so it's probably ok to keep.